tests/CommandLine.Tests/Unit/UnParserExtensionsTests.cs

```diff
@@ -3,11 +3,9 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text.RegularExpressions;
-using CommandLine.Tests.Fakes;
 using Xunit;
 using FluentAssertions;
-using Microsoft.FSharp.Core;
+using CommandLine.Tests.Fakes;
 

```

Comment:
//Microsoft.FSharp.Core is needed for fsharp Target_Build. This is the reason for appveyor fail
```suggestion
using Microsoft.FSharp.Core;
```